packages/core/language/src/shared/language-switch/language-switch.component.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LanguageSwitchComponent } from './language-switch.component';
+
+describe('LanguageSwitchComponent', () => {
+  let component: LanguageSwitchComponent;
+  let fixture: ComponentFixture<LanguageSwitchComponent>;
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [LanguageSwitchComponent]
+    }).compileComponents();
```

Comment:
Tu as des erreurs au niveau des tests, présentement Angular est en transition sur changer son moteur Zone.js de changement de détection.

Pour les tests tu devra ajouter les providers manquant: 

```suggestion
    await TestBed.configureTestingModule({
      imports: [LanguageSwitchComponent],
      providers: [
        provideZonelessChangeDetection(),
        provideMockTranslation(),
        provideHttpClientTesting()
      ]
    }).compileComponents();
```